js/navigation.js

```
@@ -107,121 +127,156 @@ class Navigation {
-        // Handle link clicks
-        document.addEventListener('click', (e) => {
-            const link = e.target.closest('a');
-            if (link && link.href && !link.href.startsWith('#') && link.href.includes(window.location.host)) {
-                e.preventDefault();
-                this.navigateWithTransition(link.href);
-            }
-        });
+            // Handle link clicks
+            document.addEventListener('click', (e) => {
+                const link = e.target.closest('a');
+                if (link && link.href && !link.href.startsWith('#') && link.href.includes(window.location.host)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)